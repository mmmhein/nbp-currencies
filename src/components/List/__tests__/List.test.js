import React from 'react';
import { create } from 'react-test-renderer';

import List from '../List';

describe('COMPONENT - List', () => {
  it('renders correctly', () => {
    const component = create(<List />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
