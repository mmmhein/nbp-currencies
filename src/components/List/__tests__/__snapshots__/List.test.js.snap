// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`COMPONENT - List renders correctly 1`] = `
.c0 {
  padding: 0;
  margin: 0;
  list-style: none;
}

<ul
  className="c0"
/>
`;
